packages/mui-material/src/Tooltip/Tooltip.js

```diff
@@ -368,10 +368,16 @@ const Tooltip = React.forwardRef(function Tooltip(inProps, ref) {
       return;
     }
 
+    // Workaround for https://github.com/facebook/react/issues/7769
+    // The autoFocus of React might trigger the event before the componentDidMount.
```

Comment:
there is no "componentDidMount" anymore

Reply:
My bad for mindlessly copying code, I should probably remove the comment. The fix should still be valid, `childNode` is will be null when mouse over event is triggered, preventing the tool tip from showing up. 

Going through previous PR, I come across this comment https://github.com/mui/material-ui/pull/12372#issuecomment-409846333, do you think your suggestion is still valid? Maybe I can explore fixing the issue like you suggested in this PR rather than using this fix?